pkg/test/resourcefixture/testdata/basic/run/v1beta1/runjob/runjob-with-port/_generated_export_runjob-with-port.golden

```diff
@@ -18,12 +18,9 @@ spec:
             cpu: 1000m
             memory: 512Mi
         startupProbe:
-          failureThreshold: 3
           httpGet:
             path: /
-            port: 8081
-          periodSeconds: 10
-          timeoutSeconds: 1
+            port: 8080
```

Comment:
It looks like the mock run job  does not handle "Port" (the field is not updated)